frontend/src/components/Signup.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+
+class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      email: '',
+      password: '',
+      confirmpassword: '',
+      showmessage: false,
```

Comment:
```suggestion
      showMessage: false,
```

same goes for the other variables